Cargo.toml

```
@@ -4,3 +4,6 @@ members = [
     "libsignal-service-actix",
     "libsignal-service-hyper",
 ]
+
+[patch.crates-io]
```

```suggestion
[patch.crates-io]
"curve25519-dalek" = { git = 'https://github.com/whisperfish/curve25519-dalek', branch = 'lizard2-update' }

[patch.'https://github.com/signalapp/curve25519-dalek']
"curve25519-dalek" = { git = 'https://github.com/whisperfish/curve25519-dalek', branch = 'lizard2-update' }
```

what do you think of this to temporarily merge this and move forward until upstream merges the fix?

---

I would be completely fine with that, I think.